packages/cli-hydrogen/src/cli/commands/hydrogen/deploy.ts

```diff
@@ -34,6 +34,11 @@ export default class Deploy extends Command {
       env: 'SHOPIFY_HYDROGEN_FLAG_DMS_ADDRESS',
       default: 'https://oxygen-dms.shopifycloud.com',
     }),
+    skipHealthCheck: Flags.boolean({
```

Comment:
In `oxygenctl` I think we just call this flag `healthCheck` and then [default it to false](https://github.com/Shopify/oxygenctl/blob/5d5d30668fbc00de9faebe672600ce6458141a62/cmd/oxygenctl/deploy.go#L121) so I think we should follow that behavior just for now.